ui/src/data_explorer/apis/index.js

```diff
@@ -0,0 +1,9 @@
+import AJAX from 'src/utils/ajax'
+
+export const writeData = async (source, db, data) => {
+  return await AJAX({
```

Comment:
except that my later note about refactoring `notify` and `errorThrown` in `WriteDataForm.js` would require the use of `return` 😅